pkg/controllers/repositoryserver/repositoryserver_controller.go

```diff
@@ -90,10 +93,11 @@ func (r *RepositoryServerReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	if err := repoServerHandler.CreateOrUpdateOwnedResources(ctx); err != nil {
 		logger.Info("Setting the CR status as 'ServerStopped' since an error occurred in create/update event")
 		repoServerHandler.RepositoryServer.Status.Progress = crkanisteriov1alpha1.ServerStopped
-		if err = r.Status().Update(ctx, repoServerHandler.RepositoryServer); err != nil {
-			return ctrl.Result{}, err
+		if uerr := r.Status().Update(ctx, repoServerHandler.RepositoryServer); uerr != nil {
+			return ctrl.Result{}, uerr
 		}
-		return ctrl.Result{}, err
+		r.Recorder.Event(repoServerHandler.RepositoryServer, corev1.EventTypeWarning, "Failed", err.Error())
+		return ctrl.Result{RequeueAfter: retryDelay}, nil
```

Comment:
IMO exponential backoff timing is better to avoid resource consumption if there are multiple CRs consistently failing since a long time. How is this backoff timing calculated? How many retries/time it takes to reach to 16m as described in the PR description?

Reply:
@PrasadG193 should we get this in?

Reply:
@PrasadG193 my bad, it was caused because of another issue. Can you reverify again?